packages/job-worker/src/playout/playout.ts

```diff
@@ -837,140 +795,140 @@ export async function onPartPlaybackStarted(context: JobContext, data: OnPartPla
 			if (!playlist.activationId) throw new Error(`Rundown Playlist "${data.playlistId}" is not active!`)
 		},
 		async (cache) => {
-			const playingPartInstance = cache.PartInstances.findOne(data.partInstanceId)
-			if (!playingPartInstance)
-				throw new Error(
-					`PartInstance "${data.partInstanceId}" in RundownPlayst "${data.playlistId}" not found!`
-				)
-
-			// make sure we don't run multiple times, even if TSR calls us multiple times
-			const isPlaying =
-				playingPartInstance.timings?.startedPlayback && !playingPartInstance.timings?.stoppedPlayback
-			if (!isPlaying) {
-				logger.debug(
-					`Playout reports PartInstance "${data.partInstanceId}" has started playback on timestamp ${new Date(
-						data.startedPlayback
-					).toISOString()}`
-				)
-
-				const playlist = cache.Playlist.doc
-
-				const rundown = cache.Rundowns.findOne(playingPartInstance.rundownId)
-				if (!rundown) throw new Error(`Rundown "${playingPartInstance.rundownId}" not found!`)
-
-				const { currentPartInstance, previousPartInstance } = getSelectedPartInstancesFromCache(cache)
-
-				if (playlist.currentPartInstanceId === data.partInstanceId) {
-					// this is the current part, it has just started playback
-					if (playlist.previousPartInstanceId) {
-						if (!previousPartInstance) {
-							// We couldn't find the previous part: this is not a critical issue, but is clearly is a symptom of a larger issue
-							logger.error(
-								`Previous PartInstance "${playlist.previousPartInstanceId}" on RundownPlaylist "${playlist._id}" could not be found.`
-							)
-						} else if (!previousPartInstance.timings?.duration) {
-							onPartHasStoppedPlaying(cache, previousPartInstance, data.startedPlayback)
-						}
-					}
+			await _onPartPlaybackStarted(context, cache, data)
+		}
+	)
+}
+async function _onPartPlaybackStarted(context: JobContext, cache: CacheForPlayout, data: OnPartPlaybackStartedProps) {
+	const playingPartInstance = cache.PartInstances.findOne(data.partInstanceId)
+	if (!playingPartInstance)
+		throw new Error(`PartInstance "${data.partInstanceId}" in RundownPlayst "${data.playlistId}" not found!`)
+
+	// make sure we don't run multiple times, even if TSR calls us multiple times
```

Comment:
Comments like this that explains what is happening instead of why it's happening is a bad practice and should be avoided.
If you feel a need to include a comment like this then that's a sign that your code isn't clean enough.

Reply:
That line does not originate from this PR, so I will not adress this here.
It is an interesting thought you bring up though, I'll make sure we talk about it in our upcoming talk.